etl/steps/data/garden/lis/2023-01-18/shared.py

```diff
@@ -126,6 +126,20 @@
         "short_unit": "",
         "numDecimalPlaces": 2,
     },
+    "share_bottom50": {
+        "title": "Share of the bottom 50%",
+        "description": "This is the income or consumption of the poorest 50% as a share of total income or consumption.",
+        "unit": "%",
+        "short_unit": "%",
+        "numDecimalPlaces": 1,
+    },
+    "share_middle40": {
+        "title": "Share of the middle 40%",
+        "description": "This is the income or consumption of the middle 40% as a share of total income or consumption.",
```

Comment:
I found this definition a bit odd (maybe it's a well-known metric, but to me it wasn't). I found a better explanation [here](https://wir2022.wid.world/category/glossary/):
"The middle 40% share is the share of income/wealth accruing to the middle 40% of the population, i.e. the population whose income/wealth lies above the median and below the top 10% income threshold."
Feel free to adapt your definition if you agree.

Reply:
You're right, I will add more details in that description, thanks!